Lib/test/test_email/test_email.py

```diff
@@ -311,6 +311,34 @@ def test_as_string_policy(self):
         g.flatten(msg)
         self.assertEqual(fullrepr, s.getvalue())
 
+    def test_nonascii_as_string_without_cte(self):
+        m = """\
```

Comment:
You can use textwrap.dedent here to indent the message.  Then wrap the content lines so the whole thing still stays under 80 characters (that isn't always possible, but looks to be so in this case).

Reply:
Ah, just copied the first inline tests I saw, sorry.